pycodestyle.py

```diff
@@ -1420,7 +1420,7 @@ def stdin_get_value():
         """Read the value from stdin."""
         return TextIOWrapper(sys.stdin.buffer, errors='ignore').read()
 
-noqa = lru_cache(512)(re.compile(r'# no(?:qa|pep8)\b', re.I).search)
+noqa = lru_cache(512)(re.compile(r'# no(?:qa|pep8)(?!:)\b', re.I).search)
```

Comment:
In which way? Do you think that somebody who is doing `noga: custom-check` expects to disable all checks?

If you have `# noqa`, without `:`, it works as it had worked until now.